Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -357,29 +411,47 @@ private extension SPAuthViewController {
         performSimperiumAuthentication()
     }
     
-    @IBAction func performLogInWithMagicLink() {
+    @IBAction func requestLogInCode() {
         guard ensureWarningsAreOnScreenWhenNeeded() else {
             return
         }
-
-        presentPasswordInterface()
         
-// TODO: Restore Mail + Code Auth Flow
-//        lockdownInterface()
-//
-//        let email = self.email
-//        controller.requestLoginEmail(username: email) { error in
-//            // TODO: 429 (Rate Limited)? push PW auth instead
-//            self.presentPasswordInterface()
-//            
-//            if let error {
-//                self.handleError(error: error)
-//            } else {
-//                SPTracker.trackUserRequestedLoginLink()
-//            }
-//            
-//            self.unlockInterface()
-//        }
+        lockdownInterface()
+
+        controller.requestLoginEmail(username: email) { error in
+// TODO: 429 (Rate Limited)? push PW auth instead
```

Comment:
To be handled in a follow up. Rate limiting errors should prob. cause the Password UI to show up